enhancements/proposals/perftune.md

```diff
@@ -0,0 +1,220 @@
+# Performance improvements using perftune.py
+
+## Summary
+
+Most of the optimizations are implemented in a perftune.py script maintained by the Scylla Core team.
+Operator will be responsible for setting up correct permissions and mounts required for optimizations running 
+in the container to land on the host machine.
+
+## Motivation
+
+### Goals
+
+* K8s Nodes running Scylla Pods are optimized using perftune.py
+* All optimizations implemented in perftune.py land on host machine
+
+### Non-Goals
+
+* Revert optimizations when DaemonSet is uninstalled
+
+## Proposal
+
+We are going to use a privileged DaemonSet responsible for running the perftune.py script on each Node having Scylla Pod.
+
+Because we are tuning cluster wide resources (Nodes), we cannot have a switch inside ScyllaCluster CRD.
+That’s why we will expose an additional non-namespaced CRD consisting of node selector and tuning options.
+Such config in future can contain more configuration/optimization options like automatic disk setup/benchmark and more.
+
+### User Stories
+
+#### Story 1
+
+As a user I have several multi-purpose K8s nodes, and a couple different types of dedicated just for Scylla.
+I would like to tune just those dedicated to Scylla of type i3.xlarge. Others shouldn't be touched because optimizations may interfere with other applications.
+
+User is able to create multiple ScyllaNodeConfig to distinguish multiple types of nodes.
+
+#### Story 2
+
+As a user I have multiple K8s environments. I would like to save resources for development env and disable optimizations.
+
+User is able to completely disable Node Tuner by removing all CRs.
```

Comment:
This story is a little confusing, it could be interpreted as if removing CRs somehow reverted the optimizations.